src/engine/websites/MangaBuddy.ts

```diff
@@ -0,0 +1,24 @@
+import { Tags } from '../Tags';
+import icon from './MangaBuddy.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';

```

Comment:
If you prefer you may use a different alias to indicate that this website is based on a common theme
e.g., `import * as MadTheme from './decorators/Common';`